api/ocm/selectors/json.go

```diff
@@ -223,3 +224,14 @@ func (j JSONSchemaSelector) Match(obj map[string]string) (bool, error) {
 	}
 	return res.Valid(), nil
 }
+
+// TODO 1: this might not work.
+// TODO 2: what about accessor2?
```

Comment:
are you done with this already? The rest LGTM

Reply:
No, I need confirmation that this makes sense there. I have no idea how this works. :D

Reply:
Nice, this point is now "fixed". :D